### PR TITLE
ci: Update fake-seam-connect

### PR DESCRIPTION
**Pull request by razor-x**

### CODE REVIEW

File: test/seam/connect/client-session-token.test.ts [hunk 1]
```diff
@@ -92,6 +92,6 @@ test('SeamHttp: updateClientSessionToken checks clientSessionToken is authorized
       instanceOf: SeamHttpApiError,
     },
   )
-  t.is(err?.statusCode, 404)
-  t.is(err?.code, 'client_session_token_not_found')
+  t.is(err?.statusCode, 401)
+  t.is(err?.code, 'unauthorized')
```
Comment (andrii-balitskyi) [inline test/seam/connect/client-session-token.test.ts:96]: Changed the assertions according to [this](https://hello-seam.slack.com/archives/C07DTP6EGBZ/p1731963972774479?thread_ts=1731528462.613209&cid=C07DTP6EGBZ)

File: package.json [hunk 2]
```diff
@@ -98,8 +98,8 @@
     "axios-retry": "^4.4.2"
   },
   "devDependencies": {
-    "@seamapi/fake-seam-connect": "1.71.0",
-    "@seamapi/types": "1.292.2",
+    "@seamapi/fake-seam-connect": "^1.72.1",
+    "@seamapi/types": "^1.294.0",
```
Comment (razor-x) [inline package.json:102]:
```suggestion
    "@seamapi/fake-seam-connect": "1.72.1",
    "@seamapi/types": "1.294.0",
```

File: package.json [hunk 3]
```diff
@@ -84,7 +84,7 @@
     "npm": ">= 9.0.0"
   },
   "peerDependencies": {
-    "@seamapi/types": "^1.292.2"
+    "@seamapi/types": "^^1.294.0"
```
Comment (razor-x) [inline package.json:87]:
```suggestion
    "@seamapi/types": "^1.294.0"
```

**Comment by andrii-balitskyi** [comment]
@razor-x I think this PR can be closed in favor of https://github.com/seamapi/javascript-http/pull/150 which updates fake-seam-connect and unskips the tests?

**Pull request closed without merging**